aea/cli/utils/click_utils.py

```diff
@@ -101,6 +108,46 @@ def convert(
             self.fail(value, param, ctx)
 
 
+class PublicIdOrHashParameter(click.ParamType):
```

Comment:
Extension of `PublicId` parameter to accept both PublicId and IPFSHash as command line arguments